oauth2client/_helpers.py

```diff
@@ -26,11 +26,6 @@
 from six.moves import urllib
 
 
-__author__ = (
-    'rafek@google.com (Rafe Kaplan)',
-    'guido@google.com (Guido van Rossum)',
-)
-
```

Comment:
Ok, if code they wrote is used in the repo seems like a very small and easy bone to throw them to put them in the contributors file. Whoever this Guido guy could probably use the acknowledgement and recognition.